R/autoplot.curve_params.R

```diff
@@ -10,19 +10,21 @@
 #' @return a [ggplot2::ggplot()] object
 #' @export
 #' @examples
+#' \donttest{
 #' library(dplyr)
 #' library(ggplot2)
+#' library(magrittr)
 #'
 #' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
 #'   filter(antigen_iso %in% c("HlyE_IgA", "HlyE_IgG")) %>%
 #'   slice(1:100, .by = antigen_iso)  %>% # Reduce dataset for the purposes of this example
 #'   autoplot()
 #'
 #' curve
-#'
+#'}
 autoplot.curve_params = function(
     object,
-    antigen_isos = object$antigen_iso %>% unique(),
+    antigen_isos = unique(object$antigen_iso),
```

Comment:
Updated style based on CRAN feedback

Reply:
I believe this was a CRAN rule, but I haven't found it to be sure. Will update if I find it.